util/MsSqlMigratorUtility/MsSqlMigratorUtility.csproj

```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net6.0</TargetFramework>
+    <Nullable>enable</Nullable>

```

Comment:
❓ Compared this with https://github.com/bitwarden/server/blob/master/util/Setup/Setup.csproj and maybe these should leave? I am noticing these in libraries but not console apps.

Reply:
I created the project from VS, so I thought it was all right. I'll try to fiddle with it.